docs/plugins.rst

```diff
@@ -36,11 +51,12 @@ the ``PLUGIN_PATHS`` list can be absolute or relative to the settings file::
 
 Where to find plugins
 =====================
+Namespace plugins can be found in the `pelican-plugins organization`_ as
+individual repositories. Legacy plugins are collected in the `pelican-plugins
+repository`_ and they will be slowly phased out to the namespace versions.
```

Comment:
[…] phased out to the namespace versions → phased out in favor of the namespace versions